app/controllers/events_controller.rb

```diff
@@ -1,3 +1,48 @@
 class EventsController < ApplicationController
-  def index; end
-end
+  def index
+    @events = Event.all
+  end
+
+  def show
+    @event = Event.all
+  end
+
+  def new
+    @event = Event.new
```

Comment:
The `new` action should create a new event instance, not find an existing event.